Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/ComponentConfig.purs

```diff
@@ -993,7 +993,8 @@ chooseYourRideConfig state = ChooseYourRide.config
   {
     rideDistance = state.data.rideDistance,
     rideDuration = state.data.rideDuration,
-    quoteList = state.data.specialZoneQuoteList
+    quoteList = state.data.specialZoneQuoteList,
+    uiConfig = state.data.config
```

Comment:
instead of passing whole `config` you can pass key which is required

Reply:
done